### ISSUE TITLE
Navigation between articles Previous and Next buttons

### ISSUE THREAD

**Issue by Giorgi625** [JoomCoder-com/JoomCCK#212]
It would be a great addition if we could have a Previous and Next buttons in full view of records. which would let us navigate between articles in same section or category.

**Comment by andiamine** [comment]
Accepted, joomla articles also has this feature. so JoomCCK need it

**Comment by Giorgi625** [comment]
Yes I really needed it on Cobalt, many clients asked for it and I hired developer to create it for Cobalt earlier and he did, but it had problems in section where I had categories. It would not navigate properly.

**Comment by andiamine** [comment]
you can share with us the code and see what the issue in it

**Comment by Giorgi625** [comment]
He was not Joomla developer so he just added custom code into php files without following Joomla structure, so I doubt it will help you, but I will search for it.
